src/tscore/runroot.cc

```diff
@@ -59,20 +43,26 @@ get_yaml_path(const std::string &path)
   std::string yaml_file;
   std::string yaml_file2;
   std::string yaml_file3;
-  if (is_directory(path.c_str())) {
-    std::string yaml_file(Layout::relative_to(path, "runroot.yaml"));
-    if (exists(yaml_file)) {
+
+  std::error_code ec;
+  auto fs = ts::file::status(ts::file::path(path), ec);
+  if (ts::file::is_dir(fs)) {
+    std::string yaml_file = Layout::relative_to(path, "runroot.yaml");
+    auto yaml_fs          = ts::file::status(ts::file::path(yaml_file), ec);
+    if (is_regular_file(yaml_fs)) {
```

Comment:
Where is `is_regular_file` declared? Is that not in `ts::file`? If so, why doesn't it need namespace qualifiers?

Reply:
That's weird. Don't know why it compiles.